### PR TITLE
Feat/tbd 5475 map r6 di mr

### PR DESCRIPTION
**Pull request by bchen-talend**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.mapr600/src/org/talend/hadoop/distribution/mapr600/MapR600Distribution.java [hunk 1]
```diff
@@ -257,10 +257,10 @@ public boolean doSupportNewHBaseAPI() {
     // return false;
     // }
     //
-//     @Override
-//     public boolean doSupportHBase() {
-//     return true;
-//     }
+    // @Override
```
Comment (pbailly) [inline main/plugins/org.talend.hadoop.distribution.mapr600/src/org/talend/hadoop/distribution/mapr600/MapR600Distribution.java]: Shouldn't we support Hbase in order to be able to use MapRDB throught Hbase components?

Comment (bchen-talend) [reply main/plugins/org.talend.hadoop.distribution.mapr600/src/org/talend/hadoop/distribution/mapr600/MapR600Distribution.java]: this is for PigComponent, and I misunderstood in the beginning too

**Comment by talend-arobu** [comment]
Please do not merge until we have the QA go.

**Comment by talend-arobu** [comment]
QA passed. No bugs detected. Please merge.

**Pull request merged**